src/us/kbase/workspace/kbase/admin/WorkspaceAdministration.java

```diff
@@ -323,7 +330,8 @@ public Object runCommand(
 			final WorkspaceIdentity params = getParams(cmd, WorkspaceIdentity.class);
 			final WorkspaceIdentifier wsi = processWorkspaceIdentifier(
 							params.getWorkspace(), params.getId());
-			final WorkspaceInformation info = ws.getWorkspaceInformationAsAdmin(wsi);
+			final WorkspaceInformation info = wsmeth.getWorkspace().
```

Comment:
sometimes GitHub's diff algorithm highlights the string that has been changed, sometimes it doesn't. It's nice to be kept on your toes.